src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.html

```diff
@@ -627,15 +625,192 @@
                         *ngIf="placementTargets !== null"
                         [ngValue]="null">-- Select a placement target --</option>
                 <option *ngFor="let placementTarget of placementTargets"
-                        [value]="placementTarget.name">{{ placementTarget.description }}</option>
+                        [value]="placementTarget.name">{{placementTarget.description }}</option>
               </select>
               <cd-help-text>
                 <span i18n>
-                  When creating a bucket, a placement target can be provided as part of the LocationConstraint to override the default placement targets from the user and zonegroup.
+                  When creating a bucket, a placement target can be provided as part of the LocationConstraint to
+                  override the default placement targets from the user and zonegroup.
```

Comment:
Probably your linter updated whole of this file otherwise , I dont see any reasons for these changes.